usr/src/lib/fm/topo/modules/common/usb/Makefile

```diff
@@ -23,6 +23,6 @@ SHAREDSRCS	= topo_port.c
 
 include ../../Makefile.plugin
 
-CPPFLAGS	+= -I../shared  -I$(SRC)/uts/intel/sys/acpi -DACPI_APPLICATION
+CPPFLAGS	+= -I../shared  -I$(SRC)/uts/common/sys/acpi -DACPI_APPLICATION
```

Comment:
This is the reason, why acpica should be in usr/src/common instead -- that is, those bits are not only needed by kernel, boot loader but also by userland.

Reply:
Yeah, I've tried to not boil the ocean too much, but `usr/src/contrib` feels like a much better home for the sources.

Reply:
Better to have it boiled in one go;)